command/ca/policy/actions/policy.go

```diff
@@ -0,0 +1,197 @@
+package actions
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/urfave/cli"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"go.step.sm/cli-utils/errs"
+	"go.step.sm/linkedca"
+
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/cli/command/ca/policy/policycontext"
+	"github.com/smallstep/cli/internal/command"
+)
+
+var provisionerFilterFlag = cli.StringFlag{
+	Name:  "provisioner",
+	Usage: `The provisioner <name>`,
+}
+
+func retrieveAndInitializePolicy(ctx context.Context, client *ca.AdminClient) (*linkedca.Policy, error) {
+
+	var (
+		policy *linkedca.Policy
+		err    error
+	)
+
+	clictx := command.CLIContextFromContext(ctx)
+	provisioner := clictx.String("provisioner")
+	reference := clictx.String("eab-key-reference")
+	keyID := clictx.String("eab-key-id")
+
+	switch {
+	case policycontext.IsAuthorityPolicyLevel(ctx):
+		policy, err = client.GetAuthorityPolicy()
+	case policycontext.IsProvisionerPolicyLevel(ctx):
+		if provisioner == "" {
+			return nil, errs.RequiredFlag(clictx, "provisioner")
+		}
+		policy, err = client.GetProvisionerPolicy(provisioner)
+	case policycontext.IsACMEPolicyLevel(ctx):
+		if provisioner == "" {
+			return nil, errs.RequiredFlag(clictx, "provisioner")
+		}
+		if reference == "" && keyID == "" {
+			return nil, errs.RequiredOrFlag(clictx, "eab-key-reference", "eab-key-id")
+		}
+		policy, err = client.GetACMEPolicy(provisioner, reference, keyID)
+	default:
+		panic("no context for policy retrieval set")
+	}
+
+	if err != nil {
+		var ae = new(ca.AdminClientError)
+		if errors.As(err, &ae) && ae.Type == "notFound" { // TODO: use constant?
```

Comment:
We don't need a pointer to a pointer here.
```suggestion
		var ae ca.AdminClientError
		if errors.As(err, &ae) && ae.Type == "notFound" { // TODO: use constant?
```

Reply:
Had to go for 

```golang
var ae *ca.AdminClientError
```

Using my VS Code linter (`errorsas`)

Reply:
If it's not initialized a pointer to a pointer makes sense.